comps/retrievers/langchain/vdms/retriever_vdms.py

```diff
@@ -0,0 +1,127 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import time
+
+from langchain_community.embeddings import HuggingFaceBgeEmbeddings, HuggingFaceHubEmbeddings
+from langchain_community.vectorstores.vdms import VDMS, VDMS_Client
+from langchain_huggingface.embeddings import HuggingFaceEndpointEmbeddings
+from langsmith import traceable
```

Comment:
Please remove the langsmith related code since it's not free for using.

Reply:
Done. removed langsmith related code and the call to the @traceable decorator in the commit that will be pushed shortly.